source/header_cell.ts

```diff
@@ -1,105 +1,60 @@
-import {Sorting} from './sorting';
-import {Filter} from './filter';
+import { SortOrder } from './sort_order';
 
 /** Header of a column of a table. */
 export class HeaderCell {
 
   /** Constructs a new HeaderCell.
     * @param name - The name to display if there is enough available space.
     * @param shortName - The name to display if there is not enough space.
-    * @param isSortable - Indicates if the column is sortable.
-    * @param sorting - The current sorting of the column.
+    * @param sortOrder - The current sorting of the column.
+    * @param onSort - Callback to sort the column. Returns true if the sort was
```

Comment:
How can a sort be unsuccessful?

Reply:
I assumed that a sort could fail as any other transaction could fail. If that's a bad assumption on my part I'll just change it so that it assumes every sort is successful.